purchase_stock_bl_date/readme/DESCRIPTION.rst

```diff
@@ -1,5 +1,12 @@
 This modules does the following:
 
 * Add bl_date in purchase order.
-* Pass bl_date to bill and recipts
-* Calulate journal entry with bl_date rate
+* Pass bl_date recipts
+
+Background
+
+In standard Odoo, the system calculates the exchange rate of journal entries based on the current date
+when generating stock valuations. This can cause inconvenience if users need to compute the rate of
+journal entries based on a custom date, especially when the currency used in the record differs from
+the company's currency. This module addresses this issue by enabling the computation of the exchange rate
+based on a user-defined date.
```

Comment:
@AungKoKoLin1997 , Kindly check following description:

This modules does the following:

* Add bl_date in purchase order.
* Pass bl_date to receipts

Background

In an operation where the purchase order is created at the same time as the receipt of goods, which is done due to frequent discrepancies between the received quantity of goods and the quantity at the time of receipt, there is a condition where the currency rate in effect at the time of creating the purchase order differs from the currency rate used when the B/L (Bill of Lading) is issued.
This becomes a problem when the user wants the price calculated in the journal entry to use the currency rate used on the B/L issuance date. 

This module is designed to address this issue by calculating the price in the journal entry based on the currency rate in effect on the specified "B/L Date" in purchase order.

Reply:
>  done due to frequent discrepancies between the received quantity of goods and the quantity at the time of receipt
Not correct.
When purchasing product, you want to apply the rate of that day (BL date) because the product belong to the company from the moment the product come in ship.
Customers want to apply the rate at the time the product are placed on the ship.　

Reply:
@kanda999  @rinaldifirdaus  Is the following correct?

Background

During the procurement process, where the purchase order and receipt of goods occur simultaneously, discrepancies can frequently arise between the received quantity of goods and the quantity recorded at the time of receipt. This can also result in a difference between the currency rate in effect at the time of creating the purchase order and the currency rate in effect when the B/L (Bill of Lading) is issued.

This issue becomes critical when the user prefers to have the price calculated in the journal entry using the currency rate prevalent on the B/L issuance date rather than current date. This is because from the moment the products are loaded onto the ship (B/L date), they belong to the company. Therefore, customers want to apply the currency rate of the day when the products are placed on the ship.